bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,110 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "mozorg/home/home.html" %}
+
+{% add_lang_files "mozorg/home/index-2016" "firefox/features/index" "firefox/sync" "firefox/whatsnew_38" "firefox/whatsnew_50" %}
+
+{% block page_css %}
+  {% stylesheet 'home-b' %}
+{% endblock %}
+
+{% block intro %}
+<div class="content" id="intro-wrapper">
+  <div id="intro-mozilla">
+    {# original strings from home.html #}
+    <div class="intro-content">
+      <h2>
+        {# L10n: line break below for visual formatting only #}
+        {{ _('Internet for people, <br>not profit.') }}
+      </h2>
+      <h3>
+      {% trans %}
+        Hi. We’re Mozilla, the proudly non-profit champions of the Internet,
+        helping to keep it healthy, open and accessible to all.
+      {% endtrans %}
+      </h3>
+    </div>{#--/.intro-content--#}
+  </div>{#--/#intro-mozilla-_#}
+
+  <div id="intro-promo">
+    <div id="intro-promo-content-wrapper" class="state-default">
+      <div class="intro-content">
+        {# different promos based on browser/FxA state #}
+
+        <div class="show-not-fx show-default">
+          {# original strings from home.html #}
+          {% if l10n_has_tag('mozorg-home-fxcopy-unleash') %}
+            <h2>{{ _('Unleash your Internet') }}</h2>
+            <p>{{ _('Set the Web free and your mind will follow.') }}</p>
+          {% else %}
+            <h2>{{ _('Browse freely.') }}</h2>
+            <p>{{ _('Discover unlimited potential, endless possibility and a wide open Web.') }}</p>
+          {% endif %}
+
+          {{ download_firefox(dom_id='download-intro', button_color='button-arrow', download_location='primary cta') }}
+        </div>{#--/.show-default --#}
+
+        {# shown for Fx desktop signed out of sync, and Fx for Android & iOS #}
+        <div class="show-fx-signed-out show-fx-mobile">
+          {# lang file: firefox/sync #}
+          <h2>{{ _('Your web, how you like it, everywhere') }}</h2>
```

Comment:
Side note: we're removing firefox/sync in another open PR, and this string is not available in the new firefox/features/sync template